### ISSUE TITLE
Silent failure

### ISSUE THREAD

**Issue by brianjimenez** [haddocking/prodigy#12]
When using the attached pdb file, prodigy silently fails in the server with the following message:

```
[!] Structure contains gaps:
A ASP1 < Fragment 0 > A THR186
A LEU187 < Fragment 1 > A LEU286
A LEU287 < Fragment 2 > A GLN425
A MET426 < Fragment 3 > A SER551
B CYS552 < Fragment 4 > B CYS567

Running Prodigy for structure test
ERROR:
```

[test.pdb.zip](https://github.com/haddocking/prodigy/files/4573828/test.pdb.zip)

**Comment by brianjimenez** [comment]
Command line interface gives a better insight of the problem:

```
[+] Reading structure file: /Users/bjimenez/workspaces/haddocking/prodigy/test.pdb
[!] Structure contains gaps:
	A ASP1 < Fragment 0 > A THR186
	A LEU187 < Fragment 1 > A LEU286
	A LEU287 < Fragment 2 > A GLN425
	A MET426 < Fragment 3 > A SER551
	B CYS552 < Fragment 4 > B CYS567

[+] Parsed structure file test (2 chains, 567 residues)
Traceback (most recent call last):
  File "/Users/bjimenez/workspaces/haddocking/prodigy/venv/bin/prodigy", line 11, in <module>
    load_entry_point('prodigy', 'console_scripts', 'prodigy')()
  File "/Users/bjimenez/workspaces/haddocking/prodigy/prodigy/predict_IC.py", line 293, in main
    prodigy.predict(distance_cutoff=cmd.distance_cutoff, acc_threshold=cmd.acc_threshold)
  File "/Users/bjimenez/workspaces/haddocking/prodigy/prodigy/predict_IC.py", line 146, in predict
    _, cmplx_sasa = execute_freesasa_api(self.structure)
  File "/Users/bjimenez/workspaces/haddocking/prodigy/prodigy/lib/freesasa_tools.py", line 177, in execute_freesasa_api
    struct = structureFromBioPDB(structure, classifier,)
  File "freesasa.pyx", line 1026, in freesasa.structureFromBioPDB
  File "freesasa.pyx", line 556, in freesasa.Structure.setRadiiWithClassifier
  File "freesasa.pyx", line 577, in freesasa.Structure.setRadii
AssertionError: Error: Radius array is <= 0 for the residue: ILE ,atom:  CD 
```

**Comment by brianjimenez** [comment]
Addressed in a635b6fa12b0b74e851de0100acc8e429f4c1ede and 8fe49c647a0c094c13148eb91832bf8c8852ef49

**Comment by brianjimenez** [comment]
Proper error message now being displayed.

<img width="1160" alt="prodigy_freesasa_error" src="https://user-images.githubusercontent.com/4330278/80957560-adca8c00-8e03-11ea-8b22-424446e9464b.png">
